src/main/java/com/zombiecastlerush/building/Castle.java

```diff
@@ -9,12 +9,12 @@
     //Ctor
     public Castle() {
         // create rooms
-        Room eastWing = new Room("East-Wing", "This room is on the East side.");
-        Room westWing = new Room("West-Wing", "This room is on the West side.");
-        Room castleHall = new Room("Castle-Hall", "This hall connects the West Wing, East Wing, the Draw Bridge and the Shop");
-        Room drawBridge = new Room("Draw-Bridge", "This is the draw bridge");
-        Room combatHall = new Room("Combat-Hall", "The hall where souls are laid to rest.");
-        Shop shop = new Shop("Shop", "Welcome to our shop!");
+        Room eastWing = new Room("East-Wing", "Another box is here. Sounds like the moans of a man in agony grow louder as you venture deeper into this room.");
+        Room westWing = new Room("West-Wing", "Eerily quiet, only a box awaits you in this chamber.");
+        Room castleHall = new Room("Castle-Hall", "It is cold, dark, and empty, save for a dimly lit, white box. ");
```

Comment:
"It is cold, dark, and empty, save for a dimly lit, white box"
I don't understand this this text.

Reply:
save for = aside from